app/lib/cashu/utils.ts

```diff
@@ -82,16 +82,21 @@ export const getWalletCurrency = (wallet: CashuWallet) => {
  * We will remove this if cashu-ts ever updates selectProofsToSend not to return send proofs that are less than the amount.
  */
 export class ExtendedCashuWallet extends CashuWallet {
-  public readonly seed: Uint8Array;
+  private seed: Uint8Array | undefined;
```

Comment:
if we are making it optional agan, can we also go back to just having getCashuWallet instead of that plus get extended wallet?